src/utils/mtev_skiplist.c

```diff
@@ -217,6 +223,11 @@ static int mtev_skiplisti_find_compare(mtev_skiplist *sl,
                                        mtev_skiplist_comparator_t comp) {
   mtev_skiplist_node *m = NULL;
   int count=0;
+
+  /* If we weren't passed a skiplist, there's nothing
+   * to compare */
+  if (!sl) return 0;
+
```

Comment:
zero means the things are equal... this should actually still crash.

Reply:
Addressed